### PR TITLE
fpga: Updated readme

### PR DESCRIPTION
**Pull request by CyrilKoe**
Updated occamy/fpga/README.md for hero boot explanations

### CODE REVIEW

File: hw/system/occamy/fpga/README.md [hunk 1]
```diff
@@ -1,4 +1,95 @@
...
+## Generating the Bitstream
+
+To create the bitstream for Occamy on the VCU128, you currently need to follow three steps.
+
+### Start by compiling HERO
+
+The hero repository contains the GCC toolchain (to build the kernel) and the LLVM toolchain (for applications).
+
+Your need to compile this at first as the bootrom will need to store `uboot-spl` given by hero.
+And the flash will need to store `uboot` which is given by hero too.
```
Comment (niwis) [inline hw/system/occamy/fpga/README.md:12]:
```suggestion
Furthermore, the flash will require `uboot` from hero.
```

**Review by niwis — APPROVED** [review]
Thanks! Just a few typos/suggestions

File: hw/system/occamy/fpga/README.md (hunk 1 above)
Comment (CyrilKoe) [reply hw/system/occamy/fpga/README.md:12]: Thanks ! ^^

**Comment by CyrilKoe** [comment]
Applied recommended changes

**Pull request merged**